druntime/src/core/sys/linux/sys/syscall.d

```diff
@@ -22,69 +22,106 @@ version (SPARC)   version = SPARC_Any;
 version (SPARC64) version = SPARC_Any;
 version (SystemZ) version = IBMZ_Any;
 
+/+
+    Check out the following links for handy tables:
+
+    - <https://gpages.juszkiewicz.com.pl/syscalls-table/syscalls.html>
+    - <https://github.com/hrw/syscalls-table>
+ +/
+
 // <asm/unistd.h>
 // List the numbers of the system calls the system knows.
 version (X86_64)
 {
     version (D_X32)
-        enum __NR_perf_event_open = 0x40000000 + 298;
+    {
+        enum __X32_SYSCALL_BIT = 0x40000000;
+
+        enum __NR_getrandom = __X32_SYSCALL_BIT + 318;
+        enum __NR_perf_event_open = __X32_SYSCALL_BIT + 298;
```

Comment:
are these really given as hex + decimal in the header instead of hex bitor hex?

Reply:
Yes.
```
unistd_x32.h:278:#define __NR_getrandom (__X32_SYSCALL_BIT + 318)
unistd_64.h:322:#define __NR_getrandom 318
unistd_32.h:355:#define __NR_getrandom 355
```